### PR TITLE
Attachments with active storage

### PR DESCRIPTION
**Pull request by TeachMeHow**
set up on hard drive for all environments

### CODE REVIEW

File: app/controllers/organizations_controller.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def create
 
     respond_to do |format|
       if @organization.save
-        format.html { redirect_to @organization, notice: 'Organization was successfully created.' }
+        format.html { redirect_to @organization, notice: 'Organizacja utworzona pomyslnie' }
```
Comment (Quintasan) [inline app/controllers/organizations_controller.rb]: Use i18n

File: app/views/applications/new.html.haml [hunk 2]
```diff
@@ -118,5 +118,7 @@
               %button.btn.btn-default.remove_row{type: "button"} Usuń
           %button.btn.btn-default#add_member{ type: "button" } Dodaj członka
     %h4.mt-5 Załączniki
+    //%input{type: "file", name: "pic", accept: "image/*", multiple:"true"}
```
Comment (Quintasan) [inline app/views/applications/new.html.haml]: Don't add commented lines

File: app/controllers/organizations_controller.rb (hunk 1 above)
Comment (TeachMeHow) [reply app/controllers/organizations_controller.rb]: i18n robi @ostpczk bodajże, niech zostanie na razie

File: app/views/layouts/application.html.haml [hunk 3]
```diff
@@ -22,4 +22,4 @@
       .row
         .col-md-12
           = yield
-  = javascript_include_tag params[:controller] if Rails.application.config.assets.find_asset("#{params[:controller]}.js")
\ No newline at end of file
+  = javascript_include_tag params[:controller]# if Rails.application.config.assets.find_asset("#{params[:controller]}.js")
```
Comment (Quintasan) [inline app/views/layouts/application.html.haml]: Do not commit commented out code

**Pull request merged**